docs/disaster-recovery.md

```diff
@@ -0,0 +1,443 @@
+ and applying them to image within the local cluster.
+
+Creation of the rbd-mirror daemon(s) is done through the
+ custom resource definitions (CRDs).
+For more information on how to set up Ceph RBDMirror CRD,
+ refer to [documentation](https://rook.io/docs/rook/master/ceph-rbd-mirror-crd.html).
+
+>:memo: **NOTE**: rbd-mirror assumes the two clusters are named differently.
+
+## Deploy the Volume Replication CRD
+
+Volume Replication Operator follows controller pattern and provides
+ extended APIs for storage disaster recovery.
+ The extended APIs are provided via Custom Resource Definition (CRD).
+
+>:bulb: For more information, please refer to the
+> [volume-replication-operator](https://github.com/csi-addons/volume-replication-operator).
+
+* Deploy the `VolumeReplicationClass` CRD
+
+```bash
+    kubectl create -f https://raw.githubusercontent.com/csi-addons/volume-replication-operator/release-v0.1/config/crd/bases/replication.storage.openshift.io_volumereplicationclasses.yaml
+```
+
+> ```bash
+> customresourcedefinition.apiextensions.k8s.io/volumereplicationclasses.replication.storage.openshift.io created
+> ```
+
+* Deploy the VolumeReplication CRD
+
+```bash
+   kubectl create -f https://raw.githubusercontent.com/csi-addons/volume-replication-operator/release-v0.1/config/crd/bases/replication.storage.openshift.io_volumereplications.yaml
+```
+
+> ```bash
+> customresourcedefinition.apiextensions.k8s.io/volumereplications.replication.storage.openshift.io created created
+> ```
+
+The VolumeReplicationClass and VolumeReplication CRDs are now created.
+
+>:bulb: **Note:** Use the latest available release for Volume Replication Operator.
+> See [releases](https://github.com/csi-addons/volume-replication-operator/branches)
+> for more information.
+
+### Add RBAC rules for Volume Replication Operator
+
+Add the below mentioned rules to `rbd-external-provisioner-runner`
+ ClusterRole in [csi-provisioner-rbac.yaml](https://github.com/ceph/ceph-csi/blob/release-v3.3/deploy/rbd/kubernetes/csi-provisioner-rbac.yaml)
+
+```yaml=61
+  - apiGroups: ["replication.storage.openshift.io"]
+    resources: ["volumereplications", "volumereplicationclasses"]
+    verbs: ["create", "delete", "get", "list", "patch", "update", "watch"]
+  - apiGroups: ["replication.storage.openshift.io"]
+    resources: ["volumereplications/finalizers"]
+    verbs: ["update"]
+  - apiGroups: ["replication.storage.openshift.io"]
+    resources: ["volumereplications/status"]
+    verbs: ["get", "patch", "update"]
+  - apiGroups: ["replication.storage.openshift.io"]
+    resources: ["volumereplicationclasses/status"]
+    verbs: ["get"]
+```
+
+### Deploy the Volume Replication Sidecar
+
+Volume Replication Operator is a kubernetes operator that provides common
+ and reusable APIs for storage disaster recovery.
+ It is based on [csi-addons/spec](https://github.com/csi-addons/spec)
+ specification and can be used by any storage provider.
```

Comment:
This is not needed here again as the explanation is done at top of VR deployment. Move this above?